src/main/java/sirius/kernel/di/PartRegistry.java

```diff
@@ -148,30 +149,39 @@ void wireClass(Class<?> clazz) {
      * Called to initialize all field of the given class in the given object
      */
     private void wireClass(Class<?> clazz, Object object) {
-        for (Field field : clazz.getDeclaredFields()) {
-            if (!Modifier.isFinal(field.getModifiers()) && (object != null
-                                                            || Modifier.isStatic(field.getModifiers()))) {
-                getParts(FieldAnnotationProcessor.class).stream()
-                                                        .filter(p -> field.isAnnotationPresent(p.getTrigger()))
-                                                        .forEach(p -> {
-                                                            try {
-                                                                field.setAccessible(true);
-                                                                p.handle(this, object, field);
-                                                            } catch (Exception e) {
-                                                                Injector.LOG.WARN(
-                                                                        "Cannot process annotation %s on %s.%s: %s "
-                                                                        + "(%s)",
-                                                                        p.getTrigger().getName(),
-                                                                        field.getDeclaringClass().getName(),
-                                                                        field.getName(),
-                                                                        e.getMessage(),
-                                                                        e.getClass().getName());
-                                                            }
-                                                        });
+        try {
+            for (Field field : clazz.getDeclaredFields()) {
+                if (!Modifier.isFinal(field.getModifiers()) && (object != null
+                                                                || Modifier.isStatic(field.getModifiers()))) {
+                    wireField(object, field);
+                }
             }
+        } catch (NoClassDefFoundError e) {
+            // These errors probably come from a missing driver or the like and can most probably be ignored...
+            Exceptions.ignore(e);
```

Comment:
Better log the exception. IMO this should only be a "quick fix" so the application can actually start at all. We should really handle these cases with proper framework handling.

Reply:
Then every startup of systems without the IBM driver will log this at every startup?